apps/client/src/app/pages/island-workshop/island-workshop/island-workshop.component.html

```diff
@@ -62,7 +66,8 @@
 <ng-container *ngIf="optimizerResult$ | async as optimizerResult">
   <app-workshop-planning *ngIf="optimizerResult.planning !== null" [planning]="optimizerResult.planning" [totalScore]="optimizerResult.score"
                          [today]="today$ | async"
-                         [weeklyReset]="previousWeeklyReset$ | async"></app-workshop-planning>
+                         [weeklyReset]="previousWeeklyReset$ | async"
+                         [displayItemMetadata]="displayItemMetadata$.value"></app-workshop-planning>
```

Comment:
Use `| async` instead of accessing sync value
```suggestion
                         [displayItemMetadata]="displayItemMetadata$ | async"></app-workshop-planning>
```